testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -36,32 +36,33 @@ impl RunLoop {
         // Initialize and start the burnchain.
         let mut burnchain = BitcoinRegtestController::new(self.config.clone());
 
-        // self.callbacks.invoke_burn_chain_initialized(&mut burnchain);
-
-        let mut burnchain_tip = burnchain.start();
-
         let is_miner = if self.config.node.miner {
             let mut keychain = Keychain::default(self.config.node.seed.clone());
             let btc_addr = BitcoinAddress::from_bytes(
                 BitcoinNetworkType::Regtest,
                 BitcoinAddressType::PublicKeyHash,
                 &Keychain::address_from_burnchain_signer(&keychain.get_burnchain_signer()).to_bytes())
                 .unwrap();
-            info!("Configured as a miner: checking if we have UTXOs at address: {}", btc_addr);
+            info!("Miner node: checking UTXOs at address: {}", btc_addr);
 
             let utxos = burnchain.get_utxos(
                 &keychain.generate_op_signer().get_public_key(), 1);
             if utxos.is_none() {
-                error!("I have no UTXOs... spawning as a follower. Restart node when you get some UTXOs.");
+                error!("Miner node: UTXOs not found. Switching to Follower node. Restart node when you get some UTXOs.");
                 false
             } else {
-                info!("Have UTXOs, starting up as miner");
+                info!("Miner node: starting up, UTXOs found.");
                 true
             }
         } else {
+            info!("Follower node: starting up");
             false
         };
 
+        // self.callbacks.invoke_burn_chain_initialized(&mut burnchain);
```

Comment:
Actually can we remove this while you're at it?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1562/commits/ba7811e182acbd7c661f8c984cd66370fe7dae6c